### PR TITLE
Update get-windowsfeature.md

### PR DESCRIPTION
**Pull request by sukihanabi**
#1106 
Step Taken
Added Notes:
Get-WindowsFeature is a Server-Manager cmdlets that are only available on servers and when the remote administration tools on a workstation is install, see Windows Server: Installing roles and features in different ways https://social.technet.microsoft.com/wiki/contents/articles/52780.windows-server-installing-roles-and-features-in-different-ways.aspx

TS:
Tried to test Get-WindowsFeature in  Windows Server 2010 R2 it works
See screenshot below:

https://snag.gy/fmuRgS.jpg
https://snag.gy/DMjK14.jpg

Verified that Get-WindowsFeature is a Server cmdlet 
References:
https://social.technet.microsoft.com/Forums/office/en-US/76189558-9da8-413f-b0ed-5b772a7edc7c/installwindowsfeature-command-does-not-work-in-ps-50-on-windows-10?forum=win10itprosetup

Alternatively, Get-WindowsOptionalFeature can be use, see reference:
https://social.technet.microsoft.com/wiki/contents/articles/52780.windows-server-installing-roles-and-features-in-different-ways.aspx

### CODE REVIEW

**Comment by e0i** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: docset/windows/Microsoft.Windows.ServerManager.Migration/get-windowsfeature.md [hunk 1]
```diff
@@ -33,6 +33,10 @@ Get-WindowsFeature [[-Name] <String[]>] [-Vhd <String>] [-ComputerName <String>]
 ## DESCRIPTION
 The **Get-WindowsFeature** cmdlet gets information about rrsandf_plural that are both available for installation and already installed on a computer that is running Windows Server 2012 R2 or an offline virtual hard disk (VHD) that is running Windows Server 2012 R2.
 
+
+> [!Note]
+> Get-WindowsFeature is a Server-Manager cmdlets that are only available on servers and when the remote administration tools on a workstation is install, see [Windows Server: Installing roles and features in different ways](https://social.technet.microsoft.com/wiki/contents/articles/52780.windows-server-installing-roles-and-features-in-different-ways.aspx)
```
Comment (JohanFreelancer9) [inline docset/windows/Microsoft.Windows.ServerManager.Migration/get-windowsfeature.md]:
```suggestion
> The Get-WindowsFeature is a Server-Manager cmdlet that is only available on servers, and only when the remote administration tools are installed on a workstation. See [Windows Server: Installing roles and features in different ways](https://social.technet.microsoft.com/wiki/contents/articles/52780.windows-server-installing-roles-and-features-in-different-ways.aspx) for more details.
```

**Comment by e0i** [comment]
@dariomws306 @get-itips @tseward

Peer review needed for this PR. Thanks.

**Comment by dariomws306** [comment]
I think we don't need to add this note the article.

It clearly states in the Synopsis:
Gets information about ### **Windows Server roles**, role services, and features that are available for installation and installed ### **on a specified server**.

Also in Description:
The Get-WindowsFeature cmdlet gets information about rrsandf_plural that are both available for installation and already installed on a computer that is running ### **Windows Server 2012 R2** or an offline virtual hard disk (VHD) that is running ### Windows Server 2012 R2.

**Comment by get-itips** [comment]
Yeah I think this is somewhat redundant if you are a Windows Server administrator and know how this works but if you are not maybe is positive to clarify? Let's see what @tseward says.

**Bonus Track:** I think is a good opportunity to fix and change the "rrsandf_plural" to "features"

**Comment by tseward** [comment]
Agreed with @dariomws306

**Comment by e0i** [comment]
cc  @sukihanabi

**Comment by sukihanabi** [comment]
@JohanFreelancer9
@tseward @dariomws306 - Note removed
@get-itips - changed the "rrsandf_plural" to "features"

Feel free to let me know of any changes needed, Thank you for your help :)

**Comment by dariomws306** [comment]
Looks good now!

**Comment by tseward** [comment]
Looks good, thanks!

**Comment by get-itips** [comment]
Ok!

**Comment by JohanFreelancer9** [comment]
Looks good.

**Comment by e0i** [comment]
@AndreaBarr The PR that has passed both copy and technical review can now be merged. Thanks.

**Pull request merged**